api/types/trustedcluster.go

```diff
@@ -309,12 +264,6 @@ func (c *TrustedClusterV2) CanChangeStateTo(t TrustedCluster) error {
 	return nil
 }
 
-// String represents a human readable version of trusted cluster settings.
-func (c *TrustedClusterV2) String() string {
-	return fmt.Sprintf("TrustedCluster(Enabled=%v,Roles=%v,Token=%v,ProxyAddress=%v,ReverseTunnelAddress=%v)",
-		c.Spec.Enabled, c.Spec.Roles, c.Spec.Token, c.Spec.ProxyAddress, c.Spec.ReverseTunnelAddress)
-}
-
```

Comment:
If this is just being removed because it conflicts with the generated `String()` method, prefer disabling the generated method with `option (gogoproto.stringer) = false` and leaving this as-is.